### ISSUE TITLE
Výdrž

### ISSUE THREAD

**Issue by dobrakmato** [pexelnet/starving#5]
Každý hráč má určitú výdrž, ktorá má nejakú úroveň. 

Rôznymi typmi fyzickej aktivity (behanie, skákanie, bojovanie,..) sa mu výdrž míňa - **20 za sekundu**. Počas nízkej fyz. aktivity sa mu výdrž dopĺňa automaticky po max. hranicu výdrže (keď je najedený rýchlejšie).

Ak má hráč nízku úroveň výdrže (< 15%), nedokáže behať.

Maximálna hranica výdrže je menšia čím viac je hráč hladný. 

Hráč si môže úroveň výdrže zvýšiť, ak sa napije tekutiny (#4) alebo zje jedlo.

**Comment by janscz** [comment]
Lze dočasně zvětšit hranici výdrže ? třeba když vypije třeba energy drink ?
Bude schopný bojovat když bude mít malou úroveň výdrže ?
Nápad: Při úplném vyčerpání by hráč "usnul" na nějaký čas dokud by se mu nezvýšila úroveň na nějakou hodnotu.

Jinak souhlasím

**Comment by dobrakmato** [comment]
Hranicu zvacsit nepojde. Bude schopny bojovat, vydrz ma byt iba vedlajsi prvok hry. A nepaci sa mi zaspavanie. Pocas spanku by mohol byt napadnuty. Ak by sme z toho spravili hlavny prvok hry, som za.

**Comment by janscz** [comment]
No myslel jsem to jako trochu realistický prvek hry, prostě když seš fakt utahanej jako pes tak asi těžko půjdeš někomu namlátil zadek že ? nebo nepoběžíš maraton

**Comment by dobrakmato** [comment]
Ak nebude moct pri unaveni ziadnu akciu vykonavat, bude staranie sa o svoju vydrz hlavnou pointou hry. Neviem ci je to spravny napad. Okrem toho, ako by to spanie prebiehalo?

**Comment by janscz** [comment]
No takto, pokud hráč dosáhne nízké úrovně tak nemůže běhat, skákat tak se mu doplňuje když bude jen tak stát, ale když bude třeba se snažit běžet nebo skákat tak se mu ten zbytek energie dojde a odpadne. Jak by to probíhalo, no v podstatě zčernání obrazovky a nemožnost dejme tomu třeba 10s cokoliv dělat.

Tzn pokud bude mít malou úroveň energie a hráč se přesto rozhodne dělat nějakou aktivitu jako třeba lézt někde tak prostě po nějaké době by dostal nějaký velký slow a vzápětí by "usnul"

**Comment by dobrakmato** [comment]
Nepaci sa mi to a myslim, ze ani hracom sa nebude pacit, ked sa rozbehnu pred zombie a nevsimu si, ze maju malu uroven vydrze a zacierni sa im obrazovka a zombie ich zabiju.

Okrem toho, ak mas nizku uroven vydrze < 15 % hra ti nepovoli zacat bezat.

**Comment by janscz** [comment]
Ok, tak to necháme teda na tom jak si to popsal, ale pokud mají výdrž velmi nízkou tak bych jim dal ještě jako bonus nějaký "movement slow"

**Comment by dobrakmato** [comment]
@pitkes22 @kubob

**Comment by pitkes22** [comment]
Suhlasim s zakladnou verziou + cim menej energie tym pomalsie behaju/Chodia

**Comment by kubob** [comment]
Ten spánok nie je dobrý nápad. Hráčom by to brutálne vadilo. BTW: Bude nejaká váha? Napr. že môžeš odniesť iba 40kg a potom ideš pomalšie, rýchlejšie míňaš staminu.

**Comment by pitkes22** [comment]
Ano, aj s batohmi a dalenim inventara ... mozem to zajtra zpisať ....

**Comment by dobrakmato** [comment]
To sa mi nepaci a bude to zlozite aj na pocitanie (nehovorim o tom, ze bude treba definovat vahu kazdeho predmetu potom). A nie som si isty ci je to delenie inventara dobry napad. Napriklad aj kvoli zlozitosti pre hraca ako aj kvoli zlozitosti pre nas (naprogramovat, pretexturovat inv - dost pravdepodobne s tym bude problem, takze chcem potom aj screen, kde to funguje podla predstavy).

@pitkes22 Kde sa tu spomina energia?

**Comment by janscz** [comment]
@dobrakmato Udělal bych přímou úměru mezi Silou (psal jsem o ní v #16 ) a výdrží. Čím více síly máš, tím máš vyšší výdrž

**Comment by dobrakmato** [comment]
@janscz Priamu umeru by som tam nepridaval, nechal by som to jednoduche. Mozno neskor, ak zvazime, ze to naozaj spravi hru zabavnejsou.

**Comment by janscz** [comment]
Ok tak zatím tu přímou úměru necháme být.

Jinak souhlasím s tím jak jsi to popsal
